docs/software-platforms/aws/working-with-volumes.md

```diff
@@ -0,0 +1,81 @@
+# Working with Storage Volumes in Lightsail for Research
+
+We recommend attaching disks (also sometimes called storage volumes) to any virtual computers you create Lightsail for Research (LSfR).
+Attaching a disk to your virtual computer will increase its storage capacity.
+
+## Quick tips about disks
+
+- You can only attach your disk to one virtual computer at a time, just like an external hard drive.
+- You can only attach a disk to a virtual computer for the first time if the virtual computer is running.
+- You can attach multiple disks to the same virtual computer.
+- You must detach a disk from a virtual computer before you can delete it.
+- Your disk usage [counts towards your monthly AWS budget](../../getting-started/accessing-resources/getting-access-to-compute.md#monthly-budget).
+
+## Creating and attaching a disk
+
+1. Navigate to the Storage section of the LSfR console and click the `Create disk` button.
+
+    <figure markdown="span">
+        ![Storage section](../../img/storage-volumes-1.png){width="850"}
+    </figure>
+
+1. Give your disk a descriptive name, select the `us-east-2` region, and pick an appropriate size for your disk.
```

Comment:
Don't forget to note they should click `Create disk` after making their selections. I know it's obvious but can't hurt to say.